### DISCUSSION TITLE
Add ungroup / regroup functionality

### DISCUSSION THREAD

**Discussion by sxweet** [Garux/netradiant-custom#85] [Ideas]
Under the Entity menu I'm missing the ungroup / regroup function that other radiant versions does have. Possible to add this feature again?

**Comment by Garux** [comment]
These were replaced by less contextual and demanding to selection commands:
```
Q: how to move brushes into group entity?
A: select brushes to move, select target entity brush, Move Primitives to Entity

Q: how to remove brushes from group entity?
A: select brushes, entities creation menu->worldspawn (right mouse button click to ungroup whole entity(ies))
```

**Comment by sxweet** [comment]
Thanks for the info. I managed to do it like you suggested.
